openmc/statepoint.py

```diff
@@ -497,17 +497,28 @@ def sparse(self, sparse):
                 self.tallies[tally_id].sparse = self.sparse
 
     def get_tally(self, scores=[], filters=[], nuclides=[],
-                  name=None, id=None, estimator=None, exact=False):
+                  name=None, id=None, estimator=None, exact_nuclides=False,
+                  exact_filters=False, exact_scores=False):
         """Finds and returns a Tally object with certain properties.
 
         This routine searches the list of Tallies and returns the first Tally
         found which satisfies all of the input parameters.
 
-        NOTE: If the "exact" parameter is False (default), the input parameters
+        If the "exactness" parameter is 0 (default), the input parameters
         do not need to match the complete Tally specification and may only
-        represent a subset of the Tally's properties. If the "exact" parameter
-        is True then the scores, filters, nuclides and estimator parameters
-        must precisely match those of any matching Tally.
+        represent a subset of the Tally's properties. If the "exactness"
+        parameter is 1 then the length of the scores, filters, nuclides
+        parameters must precisely match those of any matching Tally, but the
+        filter bins may represent a subset of those in any matching Tally
+        (useful if tallies are merged in the input). If the "exactness"
+        parameter is 2 then the values of the scores, nuclides and filters
+        parameters must precisely match those of any matching Tally.
```

Comment:
From your implementation, it doesn't look like an `exactness` parameter of 2 yields any different behavior than having a value of 1. 

Also, what do you think about having an `exact_filter_bins` parameter instead of allowing the exactness parameter to go to 2? It seems like an exactness parameter of 1 for nuclides and scores will be guaranteed to have the same behavior as an exactness parameter of 2, per your description.